fungible-token/src/lib.rs

```diff
@@ -4,20 +4,30 @@
 #![no_std]
 #![feature(const_btree_new)]
 
-use codec::{Decode, Encode};
-use gstd::{debug, exec, msg, prelude::*, ActorId};
+use fungible_token_messages::{
+    Action, ApproveReply, Event, InitConfig, TransferFromReply, TransferReply,
+};
+use gstd::{debug, msg, prelude::*, ActorId};
 use primitive_types::H256;
-use scale_info::TypeInfo;
 
-const GAS_RESERVE: u64 = 500_000_000;
+const GAS_AMOUNT: u64 = 300_000_000;
 
 #[derive(Debug)]
 struct FungibleToken {
+    /// name of the token.
```

Comment:
Rust-style doc comments are from the capital letter. 
```suggestion
    /// Name of the token.
```

And so on below.